mantle/cmd/kola/qemuexec.go

```diff
@@ -75,7 +75,7 @@ const maxAdditionalNics = 16
 
 func init() {
 	root.AddCommand(cmdQemuExec)
-	cmdQemuExec.Flags().StringVarP(&knetargs, "knetargs", "", "", "Arguments for Ignition networking on kernel commandline")
+	cmdQemuExec.Flags().StringVarP(&firstbootkargs, "firstbootkargs", "", "", "Additional first boot kernel arguments")
```

Comment:
Minor: `firstboot-kargs`?

Reply:
ehh - i'm not in love with either way. I kept no dashes since `knetargs` didn't have one either.

Reply:
If this passes CI I'll merge it.